build.gradle

```diff
@@ -224,6 +225,12 @@ project('spring-rabbit') {
 		}
 
 		compile ("ch.qos.logback:logback-classic:$logbackVersion", optional)
+
+		compile ("org.apache.logging.log4j:log4j-core:$log4j2Version", optional)
+		compile ("org.apache.logging.log4j:log4j-jcl:$log4j2Version", optional)
```

Comment:
For me that works just only basing on the Log4j 2 API: http://logging.apache.org/log4j/2.x/manual/api.html